### PR TITLE
Add security.txt, link to HackerOne

### PR DESCRIPTION
**Pull request by indirect**
The `security.txt` file is RFC-based standard for how to report security issues, and we should also link to the HackerOne site from the actual webpage.

### CODE REVIEW

**Comment by indirect** [comment]
@bundlerbot r+

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.5.1
+2.6.3
```
Comment (segiddins) [inline .ruby-version]: travis also needs to be updated?

Comment (indirect) [reply .ruby-version]: whoops, yes it does!

**Comment by indirect** [comment]
@bundlerbot r=segiddins

**Pull request merged**